src/Tools/Directory.Build.targets

```diff
@@ -0,0 +1,37 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.targets"/>
+
+  <Target Name="_GetPackageVersionInfo" DependsOnTargets="$(GetPackageVersionDependsOn)"
+          Returns="@(_ProjectPathWithVersion)">
+    <ItemGroup>
+      <_ProjectPathWithVersion Include="$(MSBuildProjectFullPath)">
+        <PackageVersion>$(Version)</PackageVersion>
+      </_ProjectPathWithVersion>
+    </ItemGroup>
+  </Target>
```

Comment:
Similar to my other comment; we should document what this is doing and why we need to do this.  (Do we need to get the version in this manner?  Do we not have an MSBuild property that gets populated elsewhere?)